blueprint/src/web.tex

```diff
@@ -15,6 +15,7 @@
 
 %\home{https://github.com/YaelDillies/LeanAPAP/}
 \github{https://github.com/YaelDillies/LeanAPAP/}
+\dochome{https://utensil.github.io/LeanAPAP/docs/}
```

Comment:
This is where to configure the url prefix of Doc-gen4 docs, change accordingly during local development.

Reply:
```suggestion
\dochome{https://YaelDillies.github.io/LeanAPAP/docs/}
```

Need to change this before merging. This was required for testing in the fork.